chatterbot/utils/module_loading.py

```diff
@@ -1,4 +1,5 @@
-import importlib
+from importlib import import_module as import_string
+from inspect import isclass
```

Comment:
It appears to be both python ~2.7 and 3. I am planning on scrapping importing data files as python modules in favor of importing them as regular files. The advantage is that it is easier to traverse a directory and find all of the contained .json files than it is to traverse a python module to find all of the sub-modules.